scripts/database-subnet-whitelisting.sh

```diff
@@ -0,0 +1,33 @@
+set -e
+
+JSONPATH="[?(contains(type, 'Microsoft.DBforPostgreSQL/servers') && contains(name, 'perftest'))].name"
+POSTGRES_LIST=$(az resource list --query "${JSONPATH}" --subscription="" -o tsv | sort -u)
```

Comment:
Is this needed? Curious - think we should be using https://docs.microsoft.com/en-us/cli/azure/postgres/server?view=azure-cli-latest#az_postgres_server_list to keep it simple

Reply:
ideally we can use az postgres cli as all the perftest postgres dbs are in DCD-CNP-QA